services/payflow-service/src/main/java/ua/sinaver/web3/payflow/data/webhooks/WebhookData.java

```diff
@@ -0,0 +1,187 @@
+package ua.sinaver.web3.payflow.data.webhooks;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+public record WebhookData(@JsonProperty("created_at") long createdAt,
+                          @JsonProperty("type") String type,
```

Comment:
if you don't need to specify @JsonProperty for every field, typically unless it's different case you want to specify, check other messages in message package

also, can you move it there, in data mostly JPA related, in message any data used for serialization/desiarlization